content/posts/data-protection/companies/index.md

```diff
@@ -15,7 +15,7 @@ Having secure accounts is probably the top priority for companies data security,
 
 ![ps4]
```

Comment:
It would be better to use a figure here. It's easier to add stuff like titles/captions to it.

```suggestion
{{<figure src="ps4.jpg" >}}
```